src/blockchain.js

```diff
@@ -137,6 +168,10 @@ class Blockchain{
         this.pendingTransactions.push(transaction);
     }
 
+    /**
+     * @param {string | CryptoJS.WordArray} address 
```

Comment:
Same thing here. Wallet addresses are always converted to strings. Never WordArray objects.